collectors/python.d.plugin/dockerd/README.md

```diff
@@ -5,6 +5,14 @@ Module monitor docker health metrics.
 **Requirement:**
 
 -   `docker` package, required version 3.2.0+
+- `netdata` user should be added to the group `docker` so it can access the socket. If you installed netdata **after** you installed docker, then the user is already added to the proper group. Otherwise, please add it manually.
```

Comment:
```suggestion
-   `netdata` user should be added to the group `docker` so it can access the socket. If you installed Netdata **after** you installed Docker, then the user is already added to the proper group. Otherwise, please add it manually.
```